etcdserver/etcdserverpb/rpc.proto

```diff
@@ -566,6 +566,17 @@ message AuthenticateRequest {
   string password = 2;
 }
 
+// What is the difference between AuthenticateRequest and InternalAuthenticateRequest?
+// InternalAuthenticateRequest has a member that is filled by etcdserver and shouldn't be user-facing.
+// For avoiding misusage the field, we have an internal version of AuthenticateRequest.
+message InternalAuthenticateRequest {
```

Comment:
can we define this in raft_internal.proto? so it will be generated in the user facing api doc.